browser/src/control/Control.FormulaBarJSDialog.js

```diff
@@ -254,10 +256,12 @@ class FormulaBar {
 
 	focusField() {
 		L.DomUtil.addClass(this.getInputField(), 'focused');
+		app.preventExit();
 	}
 
 	blurField() {
 		L.DomUtil.removeClass(this.getInputField(), 'focused');
+		app.allowExit();
 	}
```

Comment:
why do we want to allow exit independently by the event that lead to losing focus ? I mean we don't know if the blurring is due to an accepted/canceled formula.